Cargo.toml

```diff
@@ -13,18 +13,19 @@ edition = "2018"
 
 [features]
 default = [ "std" ]
-std = [ "alloc", "serde/std", "unicode-normalization/std" ]
+std = [ "alloc", "serde/std", "unicode-normalization?/std" ]
 rand = [ "crate_rand", "rand_core" ]
-alloc = [ "unicode-normalization" ]
+alloc = []
+unicode-normalization = [ "dep:unicode-normalization", "alloc" ] 
```

Comment:
doesn't work in old rust

Reply:
@elsirion I guess we don't really need to use `dep:` which is somehow recent addition.

And we can have `unicode-normalization-std` to avoid using `?`.